scripts/opensearch/Dockerfile.opensearch

```diff
@@ -1,10 +1,9 @@
 FROM opensearchproject/opensearch:${OPENSEARCH_VERSION:-latest}
 
 ARG plugin_path=/usr/share/plugins/plugin
-ARG es_path=/usr/share/opensearch
-ARG es_yml=$es_path/config/opensearch.yml
-
-RUN echo "plugins.security.disabled: true" >> $es_yml
-COPY --chown=opensearch:opensearch spec/fixtures/test_certs/* $es_path/config/test_certs/
-COPY --chown=opensearch:opensearch scripts/opensearch/opensearch-run.sh $es_path/
+ARG opensearch_path=/usr/share/opensearch
+ARG opensearch_yml=$opensearch_path/config/opensearch.yml
+ARG SECURE_INTEGRATION
 
+RUN if [ "$SECURE_INTEGRATION" != "true" ] ; then echo "plugins.security.disabled: true" >> $opensearch_yml; fi
```

Comment:
Why disable security here but uninstall in ODFE?

Reply:
When i used the same command with opendistro, i faced some problem with security plugin . Will create an issue to investigate. Since, removal worked for me without any problem, i continued with this approach.

Reply:
#27